api/valor_api/backend/core/llm_instructions_no_analysis.py

```diff
@@ -0,0 +1,560 @@
+def _generate_claims_instruction(text: str) -> str:
+    """
+    Instruction template was adapted from DeepEval's codebase https://github.com/confident-ai/deepeval/blob/main/deepeval/metrics/faithfulness/template.py.
```

Comment:
repeat: it'd be helpful to start the docstrings with a one-sentence description of what each function seeks to accomplish.